mongo/integration/client_test.go

```diff
@@ -544,62 +544,6 @@ func TestClient(t *testing.T) {
 		}, 10*time.Second)
 	})
 
-	// Test that if the minimum RTT is greater than the remaining timeout for an operation, the
-	// operation is not sent to the server and no connections are closed.
-	mt.Run("minimum RTT used to prevent sending requests", func(mt *mtest.T) {
```

Comment:
We now use 90th percentile RTT to prevent sending requests, so this test will not work. I need to add local testing to ensure 90th percentile RTT is monitored and is used to prevent sending requests. Will do that as part of GODRIVER-XXXX.

Reply:
Consider commenting out this test and linking to the new ticket. This test may be a good basis for the 90th percentile RTT test.

Reply:
Good point. Added the test back in and left a `TODO` for GODRIVER-2405 to refactor it.